functions/.nvmrc

```diff
@@ -1 +1 @@
-v10.13.0
+v10.15.1
```

Comment:
is this the new default runtime for functions?  if so, i will need to update all the azure instances

Reply:
This is just the latest v10 release.   According to the documentation, functions recommend `v10.14.1` https://docs.microsoft.com/en-us/azure/azure-functions/functions-reference-node#node-version, however is the documentation *current*?

Reply:
the document you link to was last updated yesterday, and can be tracked in github - https://github.com/MicrosoftDocs/azure-docs/blob/master/articles/azure-functions/functions-reference-node.md

i would go with their recommendation as the doc is current and maintained.

Reply:
Now set to `v10.14.1` 👍